docs/docs/indispensable.md

```diff
@@ -207,3 +207,20 @@ This module provides the following low-level addons for [Kotlin MP BigNum](https
 * `BigInteger.decodeFromAsn1ContentBytes()`
 * `encodeToAsn1Primitive()` produces an ASN.1 primitive  `BigInteger`
 * `encodeToAsn1ContentBytes()` producing the content bytes of an `Asn1Primitive` for `BigInteger`
+
+
+### Notes on Object Identifiers
+In addition to PKI-related data structures, a (rather bloated) `KnownOIDs` object is available.
+It contains all ASN.1 object identifiers from Peter Guttmann's
+[dumpasn1.cfg](https://www.cs.auckland.ac.nz/~pgut001/dumpasn1.cfg).
+Hence, handy constants such as `KnwonOIDs.ecdsaWithSHA256` are available, but also rather obscure ones such as
+`KnownOIDs.asAdjacencyAttest`.
+
+`KnownOIDs` also contains human-readable descriptions of all `KnownOIDs` constants, which tie into exposed through the `ObjectIdentifier.describe()` mechanism:
+One call to `ObjectIdentifer.Companion.describeKnownOIDs()` attaches descrptions to all OIDs contained in `KnwonOIDs` (subsequent calls are a NOOP).
+This is useful for debugging, but never called by default.
```

Comment:
do we really expect users to manually call this method? I don't like it

Reply:
alternatives? yes, you can make this quasi-automatic as part of the KnownOIDs constructor but then it will sometimes be there, sometimes not, depending on whether you referenced it or not